packages/expo-updates/e2e/__tests__/utils/bundle.ts

```diff
@@ -0,0 +1,35 @@
+import crypto from 'crypto';
+import fs from 'fs/promises';
+import path from 'path';
+
+import * as Simulator from './simulator';
+
+const EXPORT_PUBLIC_URL = 'https://expo.dev/dummy-url';
+const BUNDLE_DIST_PATH = process.env.TEST_BUNDLE_DIST_PATH;
```

Comment:
Consider renaming all the "bundle" references in this file and its filename to "update" since we want to serve the entire update.

Reply:
Makes sense -- I renamed this variable and the file to "update" as we probably will eventually want to test updates with multiple assets, but left alone the variables and method names that do specifically deal with just the bundle itself.